### DISCUSSION TITLE
Do you plan to add filters or categories?

### DISCUSSION THREAD

**Discussion by dtoecho** [trevortylerlee/astro-micro#25] [Q&A]
Astro micro is fantastic

It's the kind of blog I really want.

I want to add a filter for categories or posts in Astro Micro.

Can you help me with this?

I need your help.

**Comment by trevortylerlee** [comment]
Thanks for the kind words. I'm moving this to a discussion since it's a feature request and not an issue.

I don't have any current plans on adding categories but if that changes I'll update this thread. You can try implementing it yourself by following the [guide on the Astro Docs site](https://docs.astro.build/en/tutorial/5-astro-api/2/). 

Let me know if you need any help.
